src/utilities/place.ts

```diff
@@ -0,0 +1,138 @@
+import { Address } from "../entities/address.interface";
+import {
+  AdressComponent,
+  OpeningHourPeriods,
+  PlaceDetailApiResponse,
+  PlaceWithDetail,
+} from "../entities/PlaceApi.interface";
+import {
+  BusinessHours,
+  OpenClose,
+  Restaurant,
+} from "../entities/restaurant.interface";
+
+export function formatOpenCloseTime(time: string): string {
+  return time[0] + time[1] + ":" + time[2] + time[3];
+}
+
+export function toOpenCloseHours(
+  openTime: string,
+  closeTime: string,
+  lunchTime = "1230"
```

Comment:
So, if the hours don't fall within `lunchTime` it's not gonna return any hours? 🤔